Sources/NIO/BaseSocketChannel.swift

```diff
@@ -869,7 +869,7 @@ class BaseSocketChannel<SocketType: BaseSocketProtocol>: SelectableChannel, Chan
             return
         }
 
-        guard self.selectableEventLoop.isOpen else {
+        guard self.selectableEventLoop.acceptingRegistrations else {
```

Comment:
This rename isn't strictly necessary anymore, but I thought it was a better name. Can change it back if preferred.